tests/apidoc_login_test.go

```diff
@@ -0,0 +1,42 @@
+package tests
+
+import (
+	"encoding/json"
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/match"
+	"github.com/matrix-org/complement/internal/must"
+	"github.com/tidwall/gjson"
+	"testing"
+)
+
+func TestLogin(t *testing.T) {
+	deployment := Deploy(t, "login", b.BlueprintAlice)
+	defer deployment.Destroy(t)
+	unauthedClient := deployment.Client(t, "hs1", "")
+	t.Run("parallel", func(t *testing.T) {
+		// sytest: POST /login can log in as a user
+		t.Run("POST /login can login as user", func(t *testing.T) {
+			t.Parallel()
+			res := unauthedClient.MustDo(t, "POST", []string{"_matrix", "client", "r0", "register"}, json.RawMessage(`{
+				"auth": {
+					"type": "m.login.password"
```

Comment:
I believe this should be `m.login.dummy`

Reply:
Thanks for the review @callahad , looks like I made switched the auth types in registration and login, let me fix them asap.